Packages.props

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<ItemGroup>
+    	<PackageReference Update="BenchmarkDotNet" Version="[0.12.1]" />
```

Comment:
Probably breaks dependabot :( I like how we do it in the LMS for now.

Reply:
Visual studio will also do updates by setting the version in the projects, so people just need ot keep an eye out in reviews.  But the centralized is much nicer.